addon/lib/toolbar-button.js

```diff
@@ -68,9 +72,18 @@ function getExperimentList(availableExperiments, installedAddons) {
     }
     return experiment;
   })
-  .filter(x => x.active || !x.completed); // remove inactive, completed experiments
-
-  // Sort new experiments to the top, otherwise sort by reverse-chronological
+  // Remove inactive, completed experiments.
+  .filter(x => x.active || !x.completed)
+  // Remove experiments with a locale_grantlist, unless the user's primary
+  // locale is in the grantlist. #1961
+  .filter(x => !('locale_grantlist' in x) || x.locale_grantlist.includes(userLocale))
```

Comment:
Oh, that's a great point! You were actually really close:

```js
(x.locale_grantlist || []).includes(userLocale)
```

But...I'm just going to rewrite this to be less terse. Looking at it this morning, I think a simpler style will invite contributions from less-experienced devs, and decrease the chances of making a silly mistake due to misreading something when tired.